apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/instanceQuestion.ts

```diff
@@ -112,6 +112,9 @@ const PostBodySchema = z.union([
     submission_id: IdSchema,
     modified_at: z.string(),
     rubric_item_selected_manual: IdSchema.or(IdSchema.array())
+      // For 22+ items, qs will convert the array to an object. This converts it back.
+      // (https://github.com/ljharb/qs#parsing-arrays)
+      .or(z.record(z.string(), IdSchema).transform((val) => Object.values(val)))
```

Comment:
Rather than handling three cases, did you consider disabling qs's parsing of arrays entirely with `parseArrays: false`? That would mean we only have to handle single value and objects.

Consider leaving a comment that the decision to ignore ordering is deliberate.

Reply:
Done in e9db597dd53ac117fd001232864203e21bca6a1a.